tests/acceptance/features/webUIResharing/reshareUsers.feature

```diff
@@ -98,126 +116,159 @@ Feature: Resharing shared files with different permissions
     | field       | value               |
     | uid_owner   | user1               |
     | share_with  | user3               |
-    | file_target | /simple-folder (2)  |
+    | file_target | /Shares/simple-folder  |
     | item_type   | folder              |
     | permissions | share, delete, read |
 
   Scenario: User is allowed to reshare a file/folder with the lesser permissions, and check if it is listed for original owner
     Given user "user2" has shared folder "simple-folder" with user "user1" with "read, share, delete" permissions
+    And user "user1" has accepted the share "simple-folder" offered by user "user2"
     And user "user1" has logged in using the webUI
-    When the user shares folder "simple-folder (2)" with user "User Three" as "Advanced permissions" with permissions "delete" using the webUI
+    And the user opens folder "Shares" using the webUI
+    When the user shares folder "simple-folder" with user "User Three" as "Advanced permissions" with permissions "delete" using the webUI
     And the user re-logs in as "user2" using the webUI
     Then user "User One" should be listed as "Advanced permissions" in the collaborators list for folder "simple-folder" on the webUI
     And custom permissions "share, delete" should be set for user "User One" for folder "simple-folder" on the webUI
     And user "User Three" should be listed as "Advanced permissions" in the collaborators list for folder "simple-folder" on the webUI
     And custom permissions "delete" should be set for user "User Three" for folder "simple-folder" on the webUI
+    When user "user3" has accepted the share "simple-folder" offered by user "user1"
     And user "user3" should have received a share with these details:
     | field       | value              |
     | uid_owner   | user1              |
     | share_with  | user3              |
-    | file_target | /simple-folder (2) |
+    | file_target | /Shares/simple-folder |
     | item_type   | folder             |
     | permissions | delete, read       |
 
   Scenario: User is not allowed to reshare a file/folder with the higher permissions
     Given user "user2" has shared folder "simple-folder" with user "user1" with "read, share, delete" permissions
+    And user "user1" has accepted the share "simple-folder" offered by user "user2"
     And user "user1" has logged in using the webUI
-    When the user shares folder "simple-folder (2)" with user "User Three" as "Advanced permissions" with permissions "share, delete, update" using the webUI
+    When the user opens folder "Shares" using the webUI
+    And the user shares folder "simple-folder" with user "User Three" as "Advanced permissions" with permissions "share, delete, update" using the webUI
     Then the error message with header "Error while sharing." should be displayed on the webUI
-    And as "user3" folder "simple-folder (2)" should not exist
+    And as "user3" folder "simple-folder" should not exist
```

Comment:
I'm surprised this does not fail `user3` was created with skeleton, so `simple-folder` should exist `Shares/simple-folder` should not exist